frontend/sw360-portlet/src/main/webapp/html/homepage/mytasksubmissions/view.jsp

```diff
@@ -104,6 +104,16 @@ require(['jquery', 'modules/confirm', 'datatables.net', 'jquery-confirm'], funct
     }
 });
 
+    window.addEventListener( "pageshow", function ( event ) {
+	      var historyTraversal = event.persisted ||
```

Comment:
I do not know whether we have to attribute to the source https://stackoverflow.com/a/51649470 here